dg/media/coco/app/scripts/views/form.js

```diff
@@ -267,7 +269,33 @@ define([
             _.each(this.element_entity_map, function(entity, element) {
                 if (!this.foreign_entities[entity][element]["dependency"])
                     this.render_foreign_element(element, this.get_collection_of_element(element).toArray());
+                    
+                    if (this.entity_config.entity_name == "screening" && this.$el.find('#id_' + this.entity_config.fetch_element_that_manipulate).val() == "2"| this.$el.find('#id_' + this.entity_config.fetch_element_that_manipulate).val() == null){
+                        // hide the headers
+                        this.$el.find('th#id_age').addClass('hidden');
```

Comment:
A dictionary of list can be defined in config file with all fields/ids associated with a particular category and then they can iterated and manipulated in this file. It would ensure that, we would not have to set attributes here and can be added  in  config